source/chapters/configuration.rst

```diff
@@ -195,66 +195,68 @@ Equalizer Preferences
    :figclass: pretty-figures
 
    Equalizer Preferences
-   
+
 :menuselection:`Preferences --> Equalizer` allows you to setup the equalizers.
 
-* **Equalizer Rack**: The Equalizer Rack is a special Effect Rack, that is
-  connected to the deck's equalizer and filter controls. 
+* **Equalizer Rack**: The Equalizer Rack is a special Effect Rack that is
+  connected to the deck's equalizer and filter controls.
+
+  In this section you can select the equalizers and quick effects that are used
+  with the decks.
 
-  In this section you can select the equalizers and quick effects that used 
-  with the decks.    
+* **Equalizer Plugin**: Here you can select the effect that is used as mixing
+  EQ in each deck. By default only built-in equalizers are displayed.  If you uncheck
+  "Only allow EQ knobs to control EQ specific effects" you can select any other
+  effect.
 
-* **Equalizer Plugin**: Here you can select the effect that is uses as mixing 
-  EQ in each deck. By default only build in equalizers are displayed, but you 
-  are allowed to select any effect after you have unchecked "Only allow EQ 
-  knobs to control EQ specific effects"      
 
-  If you need different settings for each deck, you may uncheck "Use the same 
+  If you want different settings for each deck, you may uncheck "Use the same
   EQ filter for all decks"
 
-* **Quick Effect**: Here you can select the effect that is controlled by the 
-  filter knob in each deck. By default only build in filter effects are 
-  displayed in a common box for all decks, but it can be changed like above. 
+* **Quick Effect**: Here you can select the effect that is controlled by the
+  dedicated filter knob in each deck. By default only built-in filter effects are
+  selected for all decks, but that can be changed as above.
 
-* **High Shelf EQ**: This slider sets the turnover frequency of the high shelf 
-  EQ and the upper corner frequency of the mid EQ. Depending on the selected 
-  EQ type, changing this slider during playback may result in a crackling sound. 
+* **High Shelf EQ**: This slider sets the crossover frequency of the high shelf
```

Comment:
Turnover frequency is the correct term for the fequency of a shelf filter. 
Crossover frequency might be more common but is wrong in this context. 

If you want to use the term crossover, you need to write something like this: 
"This slider sets the crossover frequency between the high shelf and the mid bandpass."

By the way, the crackling is gone due to a patch. I forget about that.  
Instead it would be nice to describe a use-case when you have to adjust this slider.